src/ir/proto_to_ir.cc

```diff
@@ -117,6 +120,8 @@ void ProtoToIR::PostVisit(BlockBuilder& block,
   for (const auto& result_it : block_proto.results().values()) {
     PostVisit(result_it.second);
   }
+  // Register the module with 'SsaNames'.
+  ssa_names_.GetOrCreateID(block.GetBlockPtr());
 }
 
 void ProtoToIR::PostVisit(Operation& op,
```

Comment:
We should populate ssa_names for Operation as well. Tests are not failing currently because we have written tests in sequential order for Operations.  So, IRPrinter looks at new operation and populate ssa_names based on the size of the ssa_names operation map so far. Suppose we have a test as mentioned below, it would fail. 
`
R"(module m0 {
  block b0 {
    %1 = core.select []()
    %0 = core.merge [](<<ANY>>, <<ANY>>)
  }  // block b0
  block b1 {
    %3 = core.plus [access: "private", transform: "no"](%0, <<ANY>>)
    %2 = core.mult [access: "private", transform: "no"](%0, %1)
  }  // block b1
}  // module m0
)"
`

Reply:
Operations are being registered in the PreVisit so that the ordering is consistent with their definitions rather than usages.

Reply:
Ahh I see. Sounds good!!